tests/playbooks/kubevirt_facts_template.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Gather facts for Templates
+  hosts: localhost
+  connection: local
+  vars:
+    ansible_python_interpreter: "/usr/bin/env python"
+  tasks:
+    - name: facts for Template in default namespaces
+      k8s_facts:
+        kind: Template
+        name: working
+        namespace: default
+      register: template_fact
+    - debug:
```

Comment:
I know these are just tests, but can we use some common practise with yaml formatting, like all tasks has names, do newlines between tasks etc?